clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -652,28 +652,28 @@ and the instruction fetch ordinal.
 \code
 $ $ bin64/drrun -t drcachesim -simulator_type view -indir drmemtrace.*.dir -sim_refs 20
 Output format:
-<record#> <instr#>: T<tid> <record details>
+<--record#-> <--instr#->: <---tid---> <record details>
```

Comment:
The tid column is now confusing as there is no T in the header yet it's there in the data making it look like part of the id number or something.  Let's either put back the T in the header or remove the T from the data (I would vote for the latter).